helix-core/src/main/java/org/apache/helix/model/ClusterConfig.java

```diff
@@ -158,6 +160,7 @@
   public final static boolean DEFAULT_GLOBAL_REBALANCE_ASYNC_MODE_ENABLED = true;
   private static final int GLOBAL_TARGET_TASK_THREAD_POOL_SIZE_NOT_SET = -1;
   private static final long OFFLINE_NODE_TIME_OUT_FOR_MAINTENANCE_MODE_NOT_SET = -1;
+  private static final long OFFLINE_NODE_TIME_OUT_FOR_PURGE_NOT_SET = -1;
```

Comment:
not necessarily to be changed here, but maybe we want to have one single static var VALUE_NOT_SET = -1.